_component_design/popovers.md

```diff
@@ -14,81 +14,183 @@ last-modified: 2017-08-17
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+- Avoid opening a popover from within another popover•Open popovers via click, and never open them via hover.
+- Avoid putting important or required information in a popover because the content inside of a popover isn't always visible.
+- Use popovers to make small changes, with a small number of inputs.
+
+### Problems Being Solved
+{: .hxSubSectionTitle}
+- For users needing to make changes or take actions while remaining in their original context (page) a popover allows for small changes or actions to take place without leaving the page.
+- For user needing to be able to view additional details about an item in a workflow, a popover allows easy access to this complex or interactive information without cluttering the page.
 
 {% endcolumn %}
-{% column right:"hxCol-8" %}
-{% figure [caption:"Hover popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-hover-specs.svg){:width="100%"}
-{% endfigure %}
-{% figure [caption:"Hover popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-hover-specs2.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the popover pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-hero-image.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
-</div>
 
-## Triggered Popover
+</div>
 
+## Default Layout 
+{: .hxSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Overpopovers are triggered as a result of a user hovering over linked area in
-the UI.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The most defining aspects of a popover are the container and the pointer. The container  appears on top of the canvas. The pointer points to the popover trigger. Pointers may be used on either side of the four corners.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+- Popovers **should only be shown** when a trigger is clicked.
+- Most popovers may also have a heading and submit/cancel buttons. If buttons are not needed, a popover should close when the user clicks outside of the popover.
+- The content area may include text, inputs, or interactive content like a table or visualization.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Triggered popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered.svg){:width="100%"}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover consists of a panel, contextual pointer, heading and buttons."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-default-layout.svg){:width="80%"}
```

Comment:
These two SVGs have a couple style issues.  Mainly the headers don't match our "Container Title" font style.  I updated the file and re-exported the SVGs.  Emailed them to you.

Reply:
Same as other comment on SVG's, "Unsure what the issue with the SVG's was, but I am pushing the 5 SVGs you sent my way via Email in the newest commit to the PR."